src/Compilers/CSharp/Test/WinRT/CodeGen/WinRTCollectionTests.cs

```diff
@@ -84,7 +84,7 @@ public static void Main(string[] args)
 1
 0";
 
-            var verifier = CompileAndVerifyOnWin8Only(source, expectedOutput: expectedOutput);
+            var verifier = this.CompileAndVerifyOnWin8Only(source, expectedOutput: expectedOutput);
```

Comment:
This became an extension method and hence must be invoked with an explicit receiver.